.github/CODEOWNERS

```diff
@@ -941,12 +910,12 @@ lib/vbs @department-of-veterans-affairs/vsa-debt-resolution @department-of-veter
 lib/vetext @department-of-veterans-affairs/mobile-api-team
 lib/vic @department-of-veterans-affairs/vsp-identity
 lib/virtual_regional_office @department-of-veterans-affairs/Benefits-Team-1 @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
-lib/vre @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
+lib/vre @department-of-veterans-affairs/benefits-non-disability @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 lib/webhooks @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 modules/appeals_api @department-of-veterans-affairs/lighthouse-banana-peels
 modules/apps_api @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/lighthouse-pivot
-modules/ask_va_api @department-of-veterans-affairs/backend-review-group @department-of-veterans-affairs/ask-va-team @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
-modules/avs @department-of-veterans-affairs/backend-review-group @department-of-veterans-affairs/va-cto-health-products
+modules/ask_va_api @department-of-veterans-affairs/ask-va-team @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
+modules/avs @department-of-veterans-affairs/va-cto-health-products
```

Comment:
Please add https://github.com/orgs/department-of-veterans-affairs/teams/backend-review-group to the entry
Offending line: modules/avs @department-of-veterans-affairs/va-cto-health-products

Reply:
DONE!